src/render/index.js

```diff
@@ -116,6 +120,22 @@ class App extends React.PureComponent {
     });
   }
 
+  handleDatasetLoading(ast) {
+    const dataNodes = ast.children.filter(child => child.type === 'data');
+    if (dataNodes && dataNodes.length > 0) {
+      dataNodes.forEach(node => {
+        const { name, source } = node.properties;
+        const { content } = readFile(source.value);
```

Comment:
If we update the save functionality so that only relative path is saved, we'll need to update this so that it reads from ``${projectWorkingDirectory}/${source.value}`` or similar

Reply:
We can get the working path from  the`path` argument here  https://github.com/idyll-lang/idyll-studio/pull/69/files#diff-2a4c2e4f1d22fbd1ebf5b30dfae411b353be99274721944e1157742361c0f73aR45

Reply:
Will update!